lib/saputils.pm

```diff
@@ -31,63 +32,146 @@ SAP Applications.
 
 This package is a stateless library.
 To keep this library as generic as possible avoid as much as possible any other dependency usage,
-like other baseclass or testapi. Avoid using get_var/set_var at this level.
+like other base class or test API. Avoid using get_var/set_var at this level.
```

Comment:
testapi was right: it was referring to https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm